### DISCUSSION TITLE
PRO functionality request

### DISCUSSION THREAD

**Discussion by D0wn3r** [viewflow/viewflow#437] [General]
Deleted

**Comment by kmmbvnr** [comment]
Hi,

Thank you for using Viewflow and for your interest in expanding its functionality with Celery integration. The functionality you're looking for, including better management of asynchronous tasks and comprehensive UI support for task completion, is indeed available in the PRO version of Viewflow.

I have tailored the PRO version to handle scenarios like yours seamlessly, providing both a tighter core integration and enhanced UI capabilities. This supports not only improved functionality but also contributes to the sustainability of the open source project. If this feature is critical for your application, I strongly recommend considering an upgrade to the PRO version, since it would support the open source version as well

**Reply by D0wn3r** [reply]
Thank you for your answer.
Unfortunately, we are a small company that currently only uses Viewflow for internal processes, and we're just getting started, so the PRO version is out of budget for us.

**Reply by D0wn3r** [reply]
We are considering financing you, as we use it internally and that the free version helped us a lot, but we have some concerns we wish to have an answer from you : 
- Do you have a better documentation with the pro version ?
- Can you explain us the differents plans you have for viewflow. We would like to be able to manager job with celery and also use your FSM part

**Comment by kmmbvnr** [comment]
For all questions about PRO version please mail me directly to mikhail@viewflow.io
